config/database.yml

```diff
@@ -38,9 +39,10 @@ production:
 <% else; db_host = ENV.fetch('DB_PORT_3306_TCP_ADDR') { ENV['DB_PORT'] ? 'db' : ENV['DB_HOST'] } || '127.0.0.1' %>
 base: &default
   adapter: mysql2
+  url: <%= ENV['DATABASE_URL'] %>
   encoding: utf8
+  pool: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
```

Comment:
same as https://github.com/3scale/porta/pull/3766/files#r1624383012

Reply:
Shouldn't this be `utf8mb3` or `utf8mb3_bin` like in `schema.rb`? Just asking, I don't really know the difference between encoding, charset and collation.

https://github.com/3scale/porta/blob/43dcd03310c7d6534fabaf3c85af81b64d9a0146/db/schema.rb#L15